src/pyhf/modifiers/staterror.py

```diff
@@ -104,7 +105,8 @@ def finalize(self):
             for modifier_data in self.builder_data[modname].values():
                 modifier_data['data']['mask'] = masks[modname]
             sigmas = relerrs[masks[modname]]
-            fixed = [s == 0 for s in sigmas]
+            # list of bools, consistent with other modifiers (no numpy.bool_)
+            fixed = (sigmas == 0).tolist()
```

Comment:
```suggestion
            fixed = default_backend.tolist(sigmas == 0)
```

we can't guarantee the default_backend is numpy or has `.tolist`.

Reply:
Good catch. I'm surprised this isn't caught already in the tests.